### DISCUSSION TITLE
On batch crafting limits

### DISCUSSION THREAD

**Discussion by scarf005** [cataclysmbn/Cataclysm-BN#3687] [General]
please continue discussion for #3679 here.

**Comment by Admiral-Daala1541** [comment]
I'm still of the opinion that 200L is too low of a cap, raising it's proposed value to match or be just below the volume cap of a floor tile wouldn't be very hard and it would alleviate my concerns/fears about this making large projects much more keypress intensive.

**Comment by chaosvolt** [comment]
If I recall, 200 liters is mainly just because that's the volume limit of workbenches currently, so raising the volume limit of workbenches should be a valid option too.
